tester_app/client/app/bundles/HelloWorld/reducers/index.jsx

```diff
@@ -0,0 +1,12 @@
+// This file is our manifest of all reducers for the app.
+// See /client/app/bundles/HelloWorld/store/helloWorldStore.jsx
```

Comment:
add

// A real world app will like have many reducers and it helps to organize them in one file.